lib/sync-records.ts

```diff
@@ -1,48 +1,46 @@
 import { EmployeeType, Prisma } from "@prisma/client";
 import { upsertEmployee, validEmployeeRecords } from "./employee";
-import { getAccessToken, getRecordsByName } from "./flatfile-legacy";
 import { prismaClient } from "./prisma-client";
 import { SpaceType } from "./space";
 import { DateTime } from "luxon";
 import { createAction, ActionType } from "./action";
 import { validJobRecords, upsertJobRecords } from "./job";
-import {
-  upsertBenefitPlan,
-  upsertBenefitPlanRecords,
-  validBenefitPlanRecords,
-} from "./benefit-plan";
-import {
-  upsertEmployeeBenefitPlanRecords,
-  validEmployeeBenefitPlanRecords,
-} from "./employee-benefit-plan";
+import { upsertBenefitPlan, upsertBenefitPlanRecords } from "./benefit-plan";
+import { upsertEmployeeBenefitPlanRecords } from "./employee-benefit-plan";
+import { WorkflowType, getRecordsByName } from "./flatfile";
 
 export const syncWorkbookRecords = async ({
+  workflow,
   userId,
   organizationId,
   spaceType,
 }: {
+  workflow: WorkflowType;
   userId: string;
   organizationId: string;
   spaceType: SpaceType;
 }) => {
-  const accessToken = await getAccessToken();
-
   const employeeRecords = await getRecordsByName({
+    workflow,
     userId,
-    accessToken,
     workbookName: process.env.ONBOARDING_WORKBOOK_NAME as string,
     sheetName: "Employees",
     spaceType,
   });
   const jobRecords = await getRecordsByName({
+    workflow,
     userId,
-    accessToken,
     workbookName: process.env.ONBOARDING_WORKBOOK_NAME as string,
     sheetName: "Jobs",
     spaceType,
   });
 
-  const totalRecords = employeeRecords.length + jobRecords.length;
+  const numEmployeeRecords = employeeRecords?.length
+    ? employeeRecords.length
+    : 0;
+  const numJobRecords = jobRecords?.length ? jobRecords.length : 0;
```

Comment:
What's the return type here? Is it an empty array if it succeeds? Maybe a `jobRecords?.length ?? 0` or something would work

Reply:
?? is much cleaner here to get the number of records